plugins/services/src/js/reducers/serviceForm/JSONReducers/EnvironmentVariables.js

```diff
@@ -65,7 +65,7 @@ module.exports = {
 
     return Object.keys(state.env)
       .filter(function(key) {
-        return typeof state.env[key] === "string";
+        return state.env[key] == null || typeof state.env[key] === "string";
```

Comment:
We removed `null` why checking it here?

Reply:
This is the parser I just wanted to make sure we are catching 
```JSON
{
  "envvar": null
}
```

and
```JSON
{
  "envvar": ""
}
```

And we should not remove null ... did I miss something?

Reply:
We used to set `value: null` as a default, but you are changing this. Wonder if there are other places we could get `null`